systemtest/conftest.py

```diff
@@ -170,8 +210,19 @@ def list_crashids(self):
 
 
 @pytest.fixture
-def sqshelper(config):
-    """Generate and returns a PubSub helper using env config."""
+def queue_helper(config):
```

Comment:
This should log which one it instantiated so it's clearer from the tests what happened.